python_modules/libraries/dagster-dbt/dagster_dbt/utils.py

```diff
@@ -9,6 +10,11 @@
 ASSET_RESOURCE_TYPES = ["model", "seed", "snapshot"]
 
 
+def clean_name(name: str) -> str:
```

Comment:
Curious if we have a common `dagster._utils` for this

Reply:
It looks like we have [clean_asset_name](https://github.com/dagster-io/dagster/blob/1.10.11/python_modules/dagster/dagster/_utils/names.py) - I rename it in another PR, to include all Dagster names, and then use it in integrations.